src/main/kotlin/org/kamiblue/client/gui/hudgui/elements/combat/Armor.kt

```diff
@@ -126,6 +138,16 @@ object Armor : HudElement(
         if (itemStack.isEmpty) return
 
         RenderUtils2D.drawItem(itemStack, x, y, drawOverlay = false)
+
+        if (itemStack.isItemDamaged && colorBarMode) {
+            val health: Double = itemStack.item.getDurabilityForDisplay(itemStack)
+            val hexRgb: Int = itemStack.item.getRGBDurabilityForDisplay(itemStack)
+            val i = (13.0f - health.toFloat() * 13.0f).roundToInt()
+
+            RenderUtils2D.drawRectFilled(VertexHelper(GlStateUtils.useVbo()), Vec2d(x + 2, y + 13), Vec2d(x + 15, y + 15), ColorHolder(0, 0, 0, 255))
```

Comment:
renderHud() already has a VertexHelper as parameter. You don't need to construct new one